tests/test_suites/ShortSystemTests/test_metadata_load_legacy.sh

```diff
@@ -0,0 +1,46 @@
+timeout_set 30 seconds
+master_cfg+="MAGIC_DEBUG_LOG = $TEMP_DIR/syslog|LOG_FLUSH_ON=DEBUG"
+touch "$TEMP_DIR/syslog"
+
+CHUNKSERVERS=0 \
+	MASTERSERVERS=1 \
```

Comment:
Please consider adding a shadow and a metalogger for the final version, to check that everything is compatible.

Reply:
To properly test integration with Shadow and Metaloggers we need to mixed current binaries to legacy binaries and the testing framework doesn't support it currently. So this tests will be added as part of another PR in the future.

Reply:
Not exactly, by adding another shadow and a metalogger you will tests that this versions does not have compatibility issues even if the metadata is mfs or sfs. No need for legacy versions at this point, you are cracking the metadata on purpose.

Reply:
Not exactly, because the data flow goes from Shadows/Metalogger to Master, not the other way arround, So even when cracking the metadata on purpose, Only master will notice when loading it. Shadow and Metalogger will never know. Since Shadows will be downloading from master the already corrected metadata after master initialized. In any case i have added a Shadow and Metalogger to the test